### PR TITLE
Move the actions script to a bin directory and call then using arguments

### PR DESCRIPTION
**Pull request by oscarpolanco**
Refactor the way that the actions are handle on the repo

- Add the steps of the actions to a `bin` files
- Update the `Dockerfile` => Add the `path` of the bin folder on the image and copy the repository `bin` folder to the image
- Update the `entrypoint.sh` script to receive arguments from the actions then call the respective `bin` file 
- Update `README`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,12 +32,20 @@ action "Filter Master" {
 action "Update version" {
   needs = ["Filter Master"]
   uses = "clay/docusaurus-github-action/versions@master"
+  args="version"
```
Comment (manuelurenah) [inline README.md]:
```suggestion
  args = "version"
```

File: README.md [hunk 2]
```diff
@@ -26,12 +32,20 @@ action "Filter Master" {
...
+  env={
+      BUILD_DIR="website"
+  }
```
Comment (manuelurenah) [inline README.md:38]:
```md
env = {
  BUILD_DIR = "website"
}
```

**Review by manuelurenah — APPROVED** [review]
🏄

File: README.md (hunk 1 above)
Comment (oscarpolanco) [reply README.md]: Nice catch

File: entrypoint.sh [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+set -e
+set -u
+
+sh -c "$*"
```
Comment (pedro-rosario) [inline entrypoint.sh:6]: 🔥

**Review by pedro-rosario — APPROVED** [review]
Awesome 🔥

**Pull request merged**